docs/managing-data-quality/snowplow-event-recovery/configuration/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Configuration"
```

Comment:
This was split out from the “Concepts” page. I think the content is roughly the same.